### ISSUE TITLE
Colors inserting characters into addhost.php output

### ISSUE THREAD

**Issue by ivankisic** [librenms/librenms#5130]
LibreNMS Version: 638f61002cecc94bd48a409d3de956081e0f5656
DB Schema: 141
PHP: 7.0.11
MySQL: 5.5.50-MariaDB
RRDTool: 1.4.8
SNMP: NET-SNMP 5.7.2

When running a job to add devices using addhost.php the output seems to generate some junk which makes it hard to parse. 

```
[0;31m server-name-1 Could not connect, please check the snmp details and snmp reachability[0m
  SNMP v2c: server-name-1 No reply with community NAME
[0;31m server-name-2 Could not connect, please check the snmp details and snmp reachability[0m
  SNMP v2c: server-name-2 No reply with community NAME
```

After commenting the color line in functions.php the output is fine and I can parse it normally.

https://github.com/librenms/librenms/blob/master/includes/functions.php

`$console_color = new Console_Color2();`

I don't know enough PHP to comment whether this is normal or has to be fixed somewhere.

**Comment by laf** [comment]
Out of interest why are you parsing the output rather than using the API for adding hosts?

**Comment by laf** [comment]
No further info provided
